modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -6,28 +6,34 @@ endif::[]
+Perhaps when you are creating a Mule application you need help from someone in your Anypoint-Platform organization,
+such as a developer, who works in Studio. You can download your project as a `.jar` file and then place that file in a location from which the other person can
+import it into Studio.
 
-As the other person works on the project, the changes are applied to your project online in Flow Designer. When the other person is finished, you can open your project in Flow Designer and pick up where that person left off.
+You'll need to use *Export project sources* to share just your project source files (including all the connector credentials) without all of the other artifacts that are required by the Mule runtime.
 
-== Procedure
+As the other person works on the project, the changes are applied to your project online in Flow Designer.
+When the other person is finished, you can open your project in Flow Designer and pick up where that person left off.
 
-. In Flow Designer, follow these steps:
-.. Ensure that your project is free of errors.
-.. Click the download icon, which is to the right of the Undo and Redo arrows in the top-left corner of the screen.
-.. Select *Download Source*.
+. In Flow Designer, ensure that your project is free of errors.
+. Export your project by clicking the dots next to the Test button and select *Export* from the list.
+. Select *Export project sources* to share just your project source files, and click *Export*.
+
++
+A `.jar` file that contains your project source files, including all the connector credentials, downloads to the default location on your computer.
 +
-*Result:* A `.jar` file that contains a copy of your project is downloaded to the default location on your computer.
+// Place the .jar file in a location that the other person can access.
+
 . Have the person who works in Studio follow these steps:
-.. Enable Studio to access your organization in Anypoint Platform by selecting *Preferences* > *Anypoint Studio* > *Authentication* and entering your user ID and password for Anypoint Platform.
+.. Enable Studio to access your organization in Anypoint Platform by selecting *Preferences* > *Anypoint Studio* > *Authentication*.
 .. Import the `.jar` file by selecting *File* > *Import* and then selecting *Anypoint Studio* > *Packaged Mule Application (.jar)*.
 +
-*Result:* Studio opens the project.
-
-== What to Do Next
+Studio opens the project.
 
-The person who works in Studio can work on the project. All changes are made automatically to the project in Design Center. When the work in Studio is completed, ensure that the project is deleted from Studio. If you need help again later, repeat the procedure to share your project.
+The person who works in Studio can work on the project, and all changes are made automatically to the project in Design Center.
```

Comment:
same comment here - this is not possible today

Reply:
Fixed.